### PR TITLE
feat: implement terms acceptance

### PR DESCRIPTION
**Pull request by denis-ottofeller**
This PR is introduce real implementation of user acceptance of ironclad terms

### CODE REVIEW

File: web/src/scenes/signup/signup.tsx [hunk 1]
```diff
@@ -66,8 +68,27 @@ export function Signup() {
 
   const submit = useCallback(
     async (values: SignupFormValues) => {
-      console.log(values);
       const signup_token = localStorage.getItem("signup_token");
+      const ironcladId = crypto.randomUUID();
```
Comment (igorosip0v) [inline web/src/scenes/signup/signup.tsx:72]: @paolodamico Is it ok to have random UUID as signer id in Ironclad? 
We storing this value in Hasura.

File: web/src/scenes/signup/signup.tsx [hunk 2]
```diff
@@ -98,7 +118,7 @@ export function Signup() {
   useEffect(() => {
     const signup_token = localStorage.getItem("signup_token");
     if (!signup_token) {
-      router.push(urls.login());
+      // router.push(urls.login());
```
Comment (igorosip0v) [inline web/src/scenes/signup/signup.tsx:121]: @denis-ottofeller uncomment this

**Review by igorosip0v — CHANGES_REQUESTED** [review]

File: web/src/scenes/signup/signup.tsx [hunk 3]
```diff
@@ -66,8 +68,27 @@ export function Signup() {
...
+            throw new Error("Cannot init ironclad");
+          }
+
+          await ironClad.sendAcceptance({
+            onSuccess: resolve,
+            onError: reject,
+          });
+        });
+      } catch (err) {
+        toast.error("Something went wrong");
```
Comment (igorosip0v) [inline web/src/scenes/signup/signup.tsx:87]:
```suggestion
        toast.error("Error while accepting terms");
```

Comment (denis-ottofeller) [reply web/src/scenes/signup/signup.tsx]: We decided to rest "Something went wrong" variant in DM.

File: web/src/scenes/signup/signup.tsx (hunk 1 above)
Comment (igorosip0v) [reply web/src/scenes/signup/signup.tsx]: @paolodamico Also wanted to ask you if we have some ironclad staging secrets? to test everything

**Review by igorosip0v — APPROVED** [review]

**Pull request merged**